source/main/audio/SoundScriptManager.h

```diff
@@ -21,29 +21,30 @@
 
 #ifdef USE_OPENAL
 
-#pragma once
+    #pragma once
 
-#include "RoRPrerequisites.h"
+    #include "RoRPrerequisites.h"
+    #include "Singleton.h"
 
-#include "Singleton.h"
+    #include <OgreScriptLoader.h>
 
-#include <OgreScriptLoader.h>
+    #define SOUND_PLAY_ONCE(_ACTOR_, _TRIG_) SoundScriptManager::getSingleton().trigOnce((_ACTOR_), (_TRIG_))

```

Comment:
I'd put ` //clang-format off` around this section. It looks more readable the way it was.